lib/cisco_node_utils/snmp_notification_receiver.rb

```diff
@@ -146,11 +154,22 @@ def security
     end
 
     def vrf
-      config_get('snmp_notification_receiver', 'vrf', @name)
+      if platform == :nexus
+        config_get('snmp_notification_receiver', 'vrf', @name)
+      else
+        # get all vrf
+        all_vrf = config_get('snmp_notification_receiver', 'vrf_all')
+        all_vrf.each do |vrf|
+          bla = config_get('snmp_notification_receiver', 'vrf_values', vrf: vrf)
```

Comment:
Can we use something a bit more descriptive then `bla` :).  What kind of data is being gathered here?